cloud/aws/elasticsearch/variables.tf

```diff
@@ -0,0 +1,261 @@
+  type        = list
+  default     = []
+}
+
+variable "cluster_status_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of cluster_status detector"
+  type        = list
+  default     = []
+}
+
+variable "cluster_status_aggregation_function" {
+  description = "Aggregation function and group by for cluster_status detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ""
+}
+
+variable "cluster_status_transformation_function" {
+  description = "Transformation function for cluster_status detector (mean, min, max)"
+  type        = string
+  default     = "max"
+}
+
+variable "cluster_status_transformation_window" {
+  description = "Transformation window for cluster_status detector (i.e. 5m, 20m, 1h, 1d)"
+  type        = string
+  default     = "30m"
+}
+
+variable "cluster_status_threshold_critical" {
+  description = "Critical threshold for cluster_status detector"
+  type        = number
+  default     = 1
+}
+
+variable "cluster_status_threshold_warning" {
+  description = "Warning threshold for cluster_status detector"
+  type        = number
+  default     = 1
+}
+
+# Free_space detectors
+
+variable "free_space_disabled" {
+  description = "Disable all alerting rules for free_space detector"
+  type        = bool
+  default     = null
+}
+
+variable "free_space_disabled_critical" {
+  description = "Disable critical alerting rule for free_space detector"
+  type        = bool
+  default     = null
+}
+
+variable "free_space_disabled_warning" {
+  description = "Disable warning alerting rule for free_space detector"
+  type        = bool
+  default     = null
+}
+
+variable "free_space_notifications" {
+  description = "Notification recipients list for every alerting rules of free_space detector"
+  type        = list
+  default     = []
+}
+
+variable "free_space_notifications_warning" {
+  description = "Notification recipients list for warning alerting rule of free_space detector"
+  type        = list
+  default     = []
+}
+
+variable "free_space_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of free_space detector"
+  type        = list
+  default     = []
+}
+
+variable "free_space_aggregation_function" {
+  description = "Aggregation function and group by for free_space detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ""
+}
+
+variable "free_space_transformation_function" {
+  description = "Transformation function for free_space detector (mean, min, max)"
+  type        = string
+  default     = "min"
+}
+
+variable "free_space_transformation_window" {
+  description = "Transformation window for free_space detector (i.e. 5m, 20m, 1h, 1d)"
+  type        = string
+  default     = "15m"
+}
+
+variable "free_space_threshold_critical" {
+  description = "Critical threshold for free_space detector"
+  type        = number
+  default     = 20
```

Comment:
correct if GB but I think this could be reported as MB so `20480`

Reply:
done